docs/src/components/Markdown.css

```diff
@@ -11,11 +11,16 @@
 .Markdown code {
   background-color: var(--g-colorGray100);
   color: var(--g-colorGray300);
+  display: inline-block;
```

Comment:
@ayeshakmaz Fixes an issue where `padding` was only applied to the first line of code. The root cause was that `code` is an `inline` element by default.

## Before
![image](https://user-images.githubusercontent.com/127199/114060463-40608000-984a-11eb-9471-5aa08469aa9a.png)

## After
![image](https://user-images.githubusercontent.com/127199/114060537-55d5aa00-984a-11eb-9d62-bf8fd1ae70d3.png)

Reply:
Awesome